### PR TITLE
Changelog 2.3 release

### PR DESCRIPTION
**Pull request by ygoyal18**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+### Bug fixes
+
+- [pt-example] Pelion Edge 2.2 used protocol translator example 0.13.0, which wasn't compatible with Edge Core 0.15.0. We fixed this by upgrading the example to version 0.16.0.
+- In Pelion Edge 2.2, using the i.MX 8M Mini EVK in production mode with firmware update enabled failed with a FOTA_ASSERT after the reboot. This has been fixed.
+
+### Known issues
+
+- The Pelion Device Management portal is not correctly updated after a firmware campaign in some instances.
+- [maestro] The FeatureMgmt config resource is initialized with a maximum 3.8KB of file content. The remaining file content is truncated during initialization. This is most likely due to the limitation of the gorilla/websocket library but needs further investigation. However, you can still push a file size of a maximum of 64KB through cloud service APIs.
+- [pt-example] `cpu-temperature` device reports random values because the default CPU temperature file is not the same on Yocto and LmP.
```
Comment (AnotherButler) [inline CHANGELOG.md:33]: Are these values truly random or should we replace "random" with "incorrect"?

**Review by AnotherButler — COMMENTED** [review]
Thanks for the PR 👍 I left two questions, but they shouldn't block this from merging.

**Review by mohitsingalarm — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ygoyal18) [reply CHANGELOG.md:33]: Yes, they are random.

**Review by JanneKiiskila — DISMISSED** [review]
LGTM to me. Added the changes I wanted alread @ygoyal18 .

**Review by AnotherButler — DISMISSED** [review]

**Review by OPpuolitaival — APPROVED** [review]
Looks good

**Comment by JanneKiiskila** [comment]
This should have probably been merged, too? CHANGELOG in actual release will be the 2.2.0 based @ygoyal18 @AnotherButler . We need to copy it over to docs to the release note, too.

**Review by JanneKiiskila — APPROVED** [review]
LGTM.

**Pull request merged**